tsconfig.sdk.json

```diff
@@ -9,16 +9,9 @@
     "emitDeclarationOnly": true,
     "outDir": "./resources/embedding-sdk/dist",
     "paths": {
-      "*": [
-        "./frontend/src/*",
-        "./frontend/test/*",
-        "./enterprise/frontend/src/*",
-        "./enterprise/frontend/test/*"
-      ],
-      "cljs/*": [
-        "./target/cljs_release/*"
-      ]
-    },
+      "*": ["./frontend/src/*", "./enterprise/frontend/src/*"],
```

Comment:
This reworked https://github.com/metabase/metabase/pull/43242 by not including the test file but removing `frontend/src/metabase/visualizations/visualizations/SmartScalar/test-mocks.ts`, from being included by putting it under a `tests` folder.